bin/fuzz/fuzz_targets/grandpa-justification-parse.rs

```diff
@@ -17,6 +17,9 @@
 
 #![no_main]
 
-libfuzzer_sys::fuzz_target!(|data: &[u8]| {
-    let _ = smoldot::finality::justification::decode::decode_grandpa(data);
+libfuzzer_sys::fuzz_target!(|params: (&[u8], u8)| {
+    let _ = smoldot::finality::justification::decode::decode_grandpa(
+        params.0,
```

Comment:
shouldn't this be `params.0 + 1, usize::from(params.1)`?

Reply:
Hmmm, no

`params.0` is the data to decode, and `params.1` is the block number size. I pass `params + 1` to avoid passing 0, which intentionally panics.